src/chapters/radius-auth.xml

```diff
@@ -0,0 +1,240 @@
+                        in the JRadius implementation, it is currently broken.  Also, if you specify
+                        eap-ttls you will also need to specify the
+                        <property>radius-eap-ttls-inner-protocol</property> parameter in order to
+                        properly configure the protocol used inside the EAP TTLS tunnel.</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>radius-key-file</property></term>
+                <listitem>
+                    <para>The combination certificate and private key pair to use for TLS-based
+                        RADIUS protocols that require a client-side certificate.  The file should be
+                        present in the GUACAMOLE_HOME directory, and should be readable by the user
+                        running Tomcat (or whatever Java servlet container you're running).  If not
+                        specified, this defaults to radius.pem.</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                 <term><property>radius-key-type</property></term>
+                 <listitem>
+                     <para>The type of the key file specified by the <property>radius-key-file</property> 
+                        parameter.  If not specified, this defaults to pkcs12, the default used by 
+                        the JRadius library.</para>
+                 </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>radius-key-password</property></term>
+                <listitem>
+                    <para>The password of the private key specified in the 
+                        <property>radius-key-file</property> parameter.  Defaults to null if not specified.</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>ca-key-file</property></term>
+                <listitem>
+                    <para>The file that stores the certificate authority certificates for the
+                        connection to the RADIUS server.  Defaults to no file - if specified, the
+                        file must be present in the GUACAMOLE_HOME directory.</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>ca-key-type</property></term>
+                <listitem>
+                    <para>The type of file store used for the certificate authority.  If not
+                        specified this defaults to pem.</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>ca-key-password</property></term>
+                <listitem>
+                    <para>The password used to protect the certificate authority store, if
+                        any.  Default to null.</para>
```

Comment:
"Default to null" will not have meaning to users of the extension. The default behavior of this property needs to be described in terms not specific to the internals/code.

Reply:
Fixed.